lib/modules/datasource/azure-pipelines-tasks/index.ts

```diff
@@ -22,13 +26,53 @@ export class AzurePipelinesTasksDatasource extends Datasource {
   async getReleases({
     packageName,
   }: GetReleasesConfig): Promise<ReleaseResult | null> {
-    const versions =
-      (await this.getTasks(BUILT_IN_TASKS_URL))[packageName.toLowerCase()] ??
-      (await this.getTasks(MARKETPLACE_TASKS_URL))[packageName.toLowerCase()];
+    const platform = GlobalConfig.get('platform');
+    const endpoint = GlobalConfig.get('endpoint');
+    const { token } = hostRules.find({
+      hostType: AzurePipelinesTasksDatasource.id,
```

Comment:
I'm not sure this is set, you  might want to drop this line

Reply:
Do you mean the line to search by “hostType”? If I remove it I think it won't work, if you see the implementation of that find both the “hostType” and “url” fields have to be defined, unless I'm wrong, as I say, I'm new to TS: https://github.com/renovatebot/renovate/blob/98693e2e6797053b06b409a94f26400348e19631/lib/util/host-rules.ts#L133

Reply:
@rarkins can you answer please?